package-lock.json

```diff
@@ -3724,7 +3724,8 @@
           "version": "2.1.1",
           "resolved": false,
           "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
Our npms are producing different lockfiles here. I'm on 6.4.1 - are you newer or older than me?

Reply:
Was on 6.13.7. Updated to 6.4.1, reinstalled and pushed.

Reply:
I bevlieve your 6.13.7 is newer.. I should apparently update my npm some time xD